libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -89,19 +97,66 @@ void AC_Avoid::adjust_velocity(float kP, float accel_cmss, Vector2f &desired_vel
     // limit acceleration
     const float accel_cmss_limited = MIN(accel_cmss, AC_AVOID_ACCEL_CMSS_MAX);
 
+    const Vector2f temp = desired_vel_cms;
```

Comment:
MichaelDB in particular has an aversion to calling variables "temp" so let's rename it to something like "desired_vel_orig".